DroneInventorySystemApi/src/main/java/com/digitalojt/api/controller/PracticeController.java

```diff
@@ -0,0 +1,13 @@
+package com.digitalojt.api.controller;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class PracticeController {
+
+	@GetMapping("/hello")
```

Comment:
```suggestion
	@RequestMapping("/hello")
```
の方が良いかも。
https://midorigame-jo.com/spring-requestmapping/